docker-compose.staging.yml

```diff
@@ -0,0 +1,49 @@
+version: "3.7"
+
+# Production environment override
+services:
+  php:
+    image: 'mspserver-php-staging'
+    build:
+      context: .
+      target: php_prod
+    volumes:
+      - ./:/srv/app
+      # If you develop on Mac or Windows you can remove the vendor/ directory
+      #  from the bind-mount for better performance by enabling the next line:
+      - /srv/app/vendor
+      - /srv/app/var/cache
+    environment:
+      URL_WEB_SERVER_HOST: localhost
+      APP_SECRET: ${APP_SECRET}
+      MERCURE_JWT_SECRET: ${CADDY_MERCURE_JWT_SECRET}
+
+  caddy:
+    image: 'mspserver-caddy-staging'
+    build:
+      context: .
+      target: caddy_prod
+    environment:
+      SERVER_NAME: :80
+      MERCURE_PUBLISHER_JWT_KEY: ${CADDY_MERCURE_JWT_SECRET}
+      MERCURE_SUBSCRIBER_JWT_KEY: ${CADDY_MERCURE_JWT_SECRET}
+
+  adminer:
+    image: 'mspserver-adminer-staging'
+    build:
+      context: .
+      target: adminer_base
+    restart: always
+    environment:
+      ADMINER_DEFAULT_SERVER: database
+      ADMINER_PLUGINS: 'sql-log'
+      # dev enforced here otherwise you won't be able to log in if no password was set
+      APP_ENV: dev
```

Comment:
handy to have adminer also in this staging environment, but for adminer 'dev' must be enforced (see comment)